### PR TITLE
Scraper/howard

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/howard.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    'hops_from_seed': 1,
+                    'source_url': response.url,
+                    'source_anchor': name,
+                    'require_files': True
+                },
+                callback=self.parse_results
+            )
+
+    def parse_results(self, response):
+        for row in response.css("table tr"):
```
Comment (jodizzle) [inline osp_scraper/spiders/howard.py]: You can skip the first row, since it's the table header.

File: osp_scraper/spiders/howard.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                        'depth': 2,
+                        'hops_from_seed': 2,
+                        'source_url': response.url,
+                        'source_anchor': anchor
+                    },
+                    callback=self.parse_for_files
+                )
+
+        for item in self.parse_for_files(response):
+            yield item
```
Comment (jodizzle) [inline osp_scraper/spiders/howard.py]: I'm confused.  Why do you do a `self.parse_for_files` here but also callback to `self.parse_for_files` inside the request above?  Is this related to how some syllabi are HTML and some are PDFs?

Comment (HarrisonGregg) [inline osp_scraper/spiders/howard.py]: Yeah, exactly. Some are HTML and some are PDFs, so we need to create a PageItem for the search page, and for the HTML syllabi.

File: osp_scraper/spiders/howard.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    obvious way to identify them without performing the request.  As they are
+    unsuccessful status codes, they won't be saved.
+    """
+
+    name = "howard"
+
+    start_urls = ["https://www.howard.edu/syllabi/syllabi2/search.aspx"]
+
+    def parse(self, response):
+        options = response.css("#_ctl0_ContentPlaceHolder1_SCHOOL option")[1:]
```
Comment (HarrisonGregg) [inline osp_scraper/spiders/howard.py]: There aren't very many, but why would we want to exclude courses that aren't designated to a specific college?

File: osp_scraper/spiders/howard.py (hunk 1 above)
Comment (HarrisonGregg) [inline osp_scraper/spiders/howard.py]: I think I'd prefer not to copy the entire list with a splice if it's not required.

File: osp_scraper/spiders/howard.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        for option in options:
+            name = option.css("::text").extract_first()
+            code = option.css("::attr(value)").extract_first()
```
Comment (HarrisonGregg) [inline osp_scraper/spiders/howard.py]: Good call.

File: osp_scraper/spiders/howard.py (hunk 3 above)
Comment (jodizzle) [inline osp_scraper/spiders/howard.py]: Oops, I didn't actually mean to submit this comment.  I realized while working on the review that "No College Designated" had search results associated with it.

**Comment by HarrisonGregg** [comment]
@jodizzle How about now?

File: osp_scraper/spiders/howard.py (hunk 2 above)
Comment (jodizzle) [inline osp_scraper/spiders/howard.py]: If you could add a comment about this, that would be nice.

File: osp_scraper/spiders/howard.py (hunk 1 above)
Comment (jodizzle) [inline osp_scraper/spiders/howard.py]: It would be nice if you could avoid the first row, but I guess it's not that important.

**Pull request merged**